core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.html

```diff
@@ -36,16 +36,16 @@ <h3 class="e2e-test-settings-container" tabindex="0">Basic Settings</h3>
                   <p>
                     <mat-form-field class="e2e-test-exploration-category" appearance="fill">
                       <mat-label for="explorationCategory" class="d-block">Add a category (Choose or type new)</mat-label>
-                      <mat-select [(value)]="explorationCategoryService.displayed" class="e2e-test-exploration-category-dropdown">
+                      <mat-select [(value)]="explorationCategoryService.displayed"
+                                  (valueChange)="updateCategoryListWithUserData()"
+                                  class="e2e-test-exploration-category-dropdown">
                         <mat-option class="e2e-test-exploration-new-category-add">
                           <ngx-mat-select-search ngModel
                                                  (ngModelChange)="filterChoices($event)"
-                                                 (keydown.enter)="updateCategoryListWithUserData()"
```

Comment:
Do I understand correctly that this line means "if someone enters text and presses enter this text is treated as a category"?

Maybe let's leave it? I'd say it's expected. If it's not an obvious problem let's not fix it IMO.

Reply:
I felt like it was more intuitive if the user had to select the new category whether it be by click or by arrow keys and enter, as it felt weird when searching up for a category and hitting enter, expecting the category that was highlighted to be inputted.